wdl/Utils.wdl

```diff
@@ -758,3 +758,84 @@ task TransferVcfAnnotations {
     File annotated_vcf_index = output_file_name + ".tbi"
   }
 }
+
+
+task RenameVcfSamples {
+  input {
+    File vcf
+    File vcf_idx
+    File sample_renaming_json  # JSON with map from old sample IDs to new sample IDs
+    String samtools_cloud_docker
+    String renamed_suffix = "renamed_samples"
+    RuntimeAttr? runtime_attr_override
+  }
+
+  parameter_meta {
+      vcf: {
+        localization_optional: true
+      }
+  }
+
+  String vcf_renamed_samples_filename = basename(vcf, ".vcf.gz") + "_~{renamed_suffix}.vcf.gz"
+  String vcf_renamed_samples_index_filename = vcf_renamed_samples_filename + ".tbi"
+
+  # Disk must be scaled proportionally to the size of the VCF
+  Float input_size = size(vcf, "GiB")
+  RuntimeAttr default_attr = object {
+    mem_gb: 2.0,
+    disk_gb: ceil(10.0 + input_size),
+    cpu_cores: 1,
+    preemptible_tries: 3,
+    max_retries: 1,
+    boot_disk_gb: 10
+  }
+  RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+  command <<<
+    set -euo pipefail
+
+    # symlink vcf_index to current working dir
+    ln -s ~{vcf_idx} .
+
+    echo "Get original VCF samples"
+    export GCS_OAUTH_TOKEN=`gcloud auth application-default print-access-token`
```

Comment:
This is probably moot because I expect you will want the task removed, but I want to discuss for future reference. I wrote this in mind with local execution / non-gcp stuff in mind. The token would be empty in that case, but since the files are local, their path would be good and I expect the rest of the task would execute. Is this just too risky a strategy anyway?

Reply:
That's a fair point that it would just ignore the token in local mode, so fine to leave in. I think we had the same discussion in another PR now that I think of it.